source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,157 @@
+
+## Frequently Asked Questions
+
+### Do I need a separate certificate for each site in my organization?
+
+Nope! You can use the a single certificate to cover multiple domains spread across various environments or sites. This capability is enabled because the Global CDN uses a technology called Server Name Indication (SNI), which automatically matches inbound requests with an appropriate certificate, including custom certificates.
+
+### How long will it take to load my certificate into Pantheon?
+
+Please allow two business days to get a CSR and load the certificate.
+
+### What about sites purchased online?
+
+Custom certificates are currently available to customers with a Pantheon contract using Legacy SSL. We have no current plans to offer it to Basic or Performance sites. If you pay for your site with a credit card and are interested in using a custom certificate, please [let us know](https://learn.pantheon.io/201806-Custom-Cert-Reg.html).
+
+### Will custom certificates be self-serve?
+
+We have no current plans to offer a self-serve option. The concierge service is designed to quickly guide you through the steps required to deliver HTTPS on the Global CDN using your custom certificate, and we may follow-up with a self-serve option in the future.
+
+### Which certificates do I submit?
+
+Include the end-client certificate for your named domains, as well as the intermediate certificate, in separate files.
+
+
+### What is the maximum number of SAN entries?
+
+For the broadest client compatibility we recommend limiting the number of Subject Alternate Names to 100.
+
+
+### Are private keys available for export?
+
+Private keys are just that, private, and not available for export. They are stored securely, server side, and it’s a security best practice to not share private keys among different deployments. If you manage multiple domains, with some on Pantheon, and some outside of Pantheon, then we recommend using separate certificates, and we are happy to provide you with a new Certificate Signing Request (CSR) so we can deploy a certificate on Pantheon that only has the domains served on Pantheon.
+
+
+### What are the Global CDN IP addresses?
+
+The Global CDN currently has 4 offsets. After certificate deployment, we will provide DNS information so you can upgrade. In the examples below, `X` will be replaced with a value of `1`, `2`, `3`, or `4`:
+
+A record: `23.185.0.X`
+AAAA record 1:  `2620:12a:8000::X`
+AAAA record 2:  `2620:12a:8001::X`
+
+**Note:** AAAA records are not required, but recommended as a best practice for performance, especially for mobile devices.
```

Comment:
Is there a reason we need this disclaimer? Someone should only deploy a site without the AAAA records if they have a very good reason that isn't related to custom certificates.

Reply:
This is a very common question, so the thought is that including a brief note can ward off the question later...Thinking of it as a note, not a disclaimer...

Reply:
I've linked this note to the FAQ question in the new DNS doc about DNS managers that don't offer `AAAA` records.